"""association object CharKnow fixed again4

Revision ID: 6b01e24fbde7
Revises: 3d287f554574
Create Date: 2023-12-08 14:22:06.465787

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = '6b01e24fbde7'
down_revision = '3d287f554574'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('char_know', schema=None) as batch_op:
        batch_op.drop_constraint('char_know_subject_id_fkey', type_='foreignkey')
        batch_op.create_foreign_key(None, 'character', ['subject_id'], ['id'])

    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('char_know', schema=None) as batch_op:
        batch_op.drop_constraint(None, type_='foreignkey')
        batch_op.create_foreign_key('char_know_subject_id_fkey', 'weapon', ['subject_id'], ['id'])

    # ### end Alembic commands ###
